com.github.babluboy.nutty.json

```diff
@@ -0,0 +1,93 @@
+{
+    "id": "com.github.babluboy.nutty",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.24",
```

Comment:
Won't the application run on newer runtimes? 3.26 for example?

Reply:
There is little reason to use 3.24 over 3.28

Reply:
@TingPing thanks, updated to 3.28